lib/iris/fileformats/pp.py

```diff
@@ -1398,8 +1396,14 @@ def save(self, file_handle):
         # Before we can actually write to file, we need to calculate the header
         # elements. First things first, make sure the data is big-endian
```

Comment:
It is outside of your changes admittedly, but it seems that this comment (specifically about the endianness) may now be in the wrong place given the code that immediately follows it.